benchmarks/performance/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+find_package(benchmark CONFIG REQUIRED)
+
+add_executable(bench_performance)
+
+target_link_libraries(bench_performance
+    PRIVATE
+        benchmark::benchmark
+
+        cachemere
+)
+
+
+target_sources(bench_performance
+    PRIVATE
+    src/bench_cache.cpp
+    src/main.cpp
```

[detail] Missing indent

```suggestion
        src/bench_cache.cpp
        src/main.cpp
```